testing/test_config.py

```diff
@@ -1405,6 +1405,51 @@ def test_with_specific_inifile(
         assert inipath == p
         assert ini_config == {"x": "10"}
 
+    @pytest.mark.parametrize(
+        "tree,ini_file,args, expected_root",
```

Comment:
Let's use spaces here for readability:

```suggestion
        "tree, ini_file, args, expected_root",
```